protractor.conf.ts

```diff
@@ -150,10 +150,10 @@ exports.config = {
     },
 
     onPrepare() {
-        afterEachAllRewrite();
-        beforeEachAllRewrite();
-        afterAllRewrite();
-        beforeAllRewrite();
+        // afterEachAllRewrite();
```

Comment:
commented out code?

Reply:
I will restore it :) forgot it during some test

Reply:
I will restore it :) forgot it during some test